### PR TITLE
finished the appointment stubs

### PR DESCRIPTION
**Pull request by Jason337**

### CODE REVIEW

File: app/controllers/appointment_controller.js [hunk 1]
```diff
@@ -0,0 +1,612 @@
+//  === Requirements ===
+const { fake } = require('faker')
```
Comment (GarvieMichael) [inline app/controllers/appointment_controller.js]: Why is this added? This seems unnecessary.

File: app/controllers/appointment_controller.js [hunk 2]
```diff
@@ -0,0 +1,612 @@
...
+        return {
+            id: request.params.slot_id,
+            routeDetails: {
+                controller: 'Appointment',
+                action: 'destroy_time_slot'
+            },
+            code: faker.random.number(),
+            reason: "string",
+            message: "string",
+            status: "destroy_time_slot entry destroyed successfully",
```
Comment (GarvieMichael) [inline app/controllers/appointment_controller.js]:
```suggestion
            status: "Time slot destroyed successfully",
```

File: config/routes/agreement_management_routes.js [hunk 3]
```diff
@@ -4,8 +4,20 @@ var agreement_management_routes = express.Router()
...
+  response.send( account_management_controller.create_party(request) )
+})
+agreement_management_routes.get('/appointment/:id', (request, response) => {
+  response.send( account_management_controller.show_party(request) )
+})
+agreement_management_routes.patch('/appointment/:id', (request, response) => {
+  response.send( account_management_controller.update_party(request) )
+})
+agreement_management_routes.delete('/appointment/:id', (request, response) => {
+  response.send( account_management_controller.destroy_party(request) )
```
Comment (GarvieMichael) [inline config/routes/agreement_management_routes.js]: Shouldn't these all be going to agreement_management_controller instead of account_management_controller?

**Review by GarvieMichael — CHANGES_REQUESTED** [review]

File: app/controllers/appointment_controller.js (hunk 2 above)
Comment (Jason337) [reply app/controllers/appointment_controller.js]: done

File: config/routes/agreement_management_routes.js (hunk 3 above)
Comment (Jason337) [reply config/routes/agreement_management_routes.js]: Wow big mistake it was also only meant to be one request. Fixed now

File: app/controllers/appointment_controller.js (hunk 1 above)
Comment (Jason337) [reply app/controllers/appointment_controller.js]: removed

File: app/controllers/appointment_controller.js [hunk 4]
```diff
@@ -0,0 +1,612 @@
...
+                    postCode: faker.random.number(),
+                    city: "Paris",
+                    country: "France" 
+                }
+            },
+            relatedEntity: [
+                {
+                    id: faker.random.uuid(),
+                    href: faker.internet.url(),
+                    name: "Internet connexion problem to solve",
```
Comment (GarvieMichael) [inline app/controllers/appointment_controller.js]:
```suggestion
                    name: "Internet connection problem to solve",
```

There are a few other places that this comes up, please check for them. Sorry this is nitpicky

File: app/controllers/appointment_controller.js [hunk 5]
```diff
@@ -0,0 +1,611 @@
...
+                        postCode: faker.random.number(),
+                        city: "Paris",
+                        country: "France" 
+                    }
+                },
+                relatedEntity: [
+                    {
+                        id: faker.random.uuid(),
+                        href: faker.internet.url(),
+                        name: "Internet connexion problem to solve",
```
Comment (GarvieMichael) [inline app/controllers/appointment_controller.js]:
```suggestion
                        name: "Internet connection problem to solve",
```
There are others that use relatedEntity. Please could you check them for me.

**Review by GarvieMichael — CHANGES_REQUESTED** [review]

File: app/controllers/appointment_controller.js (hunk 4 above)
Comment (Jason337) [reply app/controllers/appointment_controller.js]: Fixed

File: app/controllers/appointment_controller.js (hunk 5 above)
Comment (Jason337) [reply app/controllers/appointment_controller.js]: done

**Review by GarvieMichael — APPROVED** [review]
